config/settings/base.py

```diff
@@ -196,7 +196,8 @@
 ADMIN_URL = "admin/"
 # https://docs.djangoproject.com/en/dev/ref/settings/#admins
 ADMINS = [
-    ("""Gnosis""", "dev@gnosis.pm"),
+    ("uxio.fuentefria", "uxio.fuentefria@gnosis.pm"),
+    ("richard", "richard.meissner@gnosis.pm"),
```

Comment:
I think this is better to be set on an environment variable.

With the amount of public usage by different orgs this is definitely something that admins would configure.

Reply:
You are right, what I would do it's make it an environment variable but leave ourselves as the default ones:
- We already had gnosis configuration in place in that variable
- Removing us would imply modifying 12 deployments in production + 6 in staging

Reply:
Done 👍🏻